### PR TITLE
readmine might throw null reference exception

### PR DESCRIPTION
**Pull request by ansyral**

### CODE REVIEW

**Comment by hellosnow** [comment]
:shipit:

**Comment by qinezh** [comment]
:+1:

File: src/Microsoft.DocAsCode.Common/YamlMime.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public static string ReadMime(TextReader reader)
                 throw new ArgumentNullException(nameof(reader));
             }
             var line = reader.ReadLine();
-            if (!line.StartsWith("#"))
+            if (line == null || !line.StartsWith("#"))
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Common/YamlMime.cs]: >. [](start = 37, length = 1)

?.

**Comment by vwxyzh** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Common/YamlMime.cs (hunk 1 above)
Comment (ansyral) [inline src/Microsoft.DocAsCode.Common/YamlMime.cs]: the return value for `?.` is `bool?`

**Pull request merged**